File: ios/VMEMapViewManager.m
//
//  VMEMapViewManager.m
//  example
//
//  Created by Sharif Fahes on 11/08/2022.
//
#import "React/RCTViewManager.h"
@interface RCT_EXTERN_MODULE(VMEMapViewManager, RCTViewManager)
RCT_EXPORT_VIEW_PROPERTY(showOverlay, BOOL)
RCT_EXPORT_VIEW_PROPERTY(displayPromptToDownloadMap, BOOL)
RCT_EXPORT_VIEW_PROPERTY(mapHash, NSString)
@end

File: ios/VMEMapViewManager.swift
//
//  VMEMapViewManager.swift
//  example
//
//  Created by Sharif Fahes on 11/08/2022.
//

import Foundation
import VisioMoveEssential
import React

@objc(VMEMapViewManager)
class VMEMapViewManager: RCTViewManager {
  var mapViewFrame: VMEMapViewFrame!
  var mapView: VMEMapView!
  
  override static func requiresMainQueueSetup() -> Bool {
    return true
  }
  
  override func view() -> UIView! {
    mapViewFrame = VMEMapViewFrame.init()
    mapView = mapViewFrame.getVMEMapView()
    return mapViewFrame
  }
  
  
  
}
